response_operations_ui/views/case.py

```diff
@@ -61,3 +73,11 @@ def update_response_status(ru_ref):
 
     return redirect(url_for('reporting_unit_bp.view_reporting_unit', ru_ref=ru_ref,
                     survey=short_name, period=period))
+
+
+def get_timestamp_for_completed_case_event(case_id):
+    complete_categories = ['OFFLINE_RESPONSE_PROCESSED', 'SUCCESSFUL_RESPONSE_UPLOAD', 'COMPLETED_BY_PHONE']
```

Comment:
For consistency this should be defined at the top of the file or `COMPLETE_STATE` should be defined in it's function